app/bundles/EmailBundle/EventListener/StatsSubscriber.php

```diff
@@ -13,26 +13,24 @@
 
 use Doctrine\ORM\EntityManager;
 use Mautic\CoreBundle\EventListener\CommonStatsSubscriber;
+use Mautic\CoreBundle\Security\Permissions\CorePermissions;
 
-/**
- * Class StatsSubscriber.
- */
 class StatsSubscriber extends CommonStatsSubscriber
 {
     /**
-     * StatsSubscriber constructor.
-     *
-     * @param EntityManager $em
+     * @param CorePermissions $security
+     * @param EntityManager   $entityManager
      */
-    public function __construct(EntityManager $em)
+    public function __construct(CorePermissions $security, EntityManager $entityManager)
     {
-        $this->repositories[]             = $repo             = $em->getRepository('MauticEmailBundle:StatDevice');
+        parent::__construct($security, $entityManager);
+        $this->repositories[]             = $repo             = $entityManager->getRepository('MauticEmailBundle:StatDevice');
         $devicesTable                     = $repo->getTableName();
         $this->permissions[$devicesTable] = [
             'stat.lead' => 'lead:leads',
         ];
 
-        $this->repositories[]       = $repo       = $em->getRepository('MauticEmailBundle:Stat');
+        $this->repositories[]       = $repo       = $entityManager->getRepository('MauticEmailBundle:Stat');
```

Comment:
should this be `$entityManager->getRepository(Stat::class)`

Reply:
https://github.com/mautic/mautic/pull/8148/commits/6e010799a850e014c76434917024dd607c36dfac